vars/npmRelease.groovy

```diff
@@ -10,12 +10,13 @@ def call(body) {
     def gitUserName = config.gitUserName ?: 'fabric8-release'
     def branch = config.branch
 
-    sh "git config user.email ${gitEmail}"
-    sh "git config user.name ${gitUserName}"
+    sh """
+       git config user.email ${gitEmail}
+       git config user.name ${gitUserName}
 
-    sh 'chmod 600 /root/.ssh-git/ssh-key'
-    sh 'chmod 600 /root/.ssh-git/ssh-key.pub'
-    sh 'chmod 700 /root/.ssh-git'
+       install -m 600 -D /root/.ssh-git-ro/ssh-key /root/.ssh-git/ssh-key
```

Comment:
@jaseemabid Are we sure we need these keys? We already have the token for Github, the token allows us to commit, create PR and other things. Why do we need this key ?

Reply:
I've no idea about the potential paths the code could take before it gets here. There is no harm in setting up both token and ssh key. We are playing a very defensive, least effort required approach with fpl nowadays.

Reply:
Okay.